examples/qc-checks/check-for-price-outliers/outliers-using-iqr.sql

```diff
@@ -0,0 +1,40 @@
+-- ============================================================
+-- Description:
+-- This query shows you how to check for outliers in a CSV
+-- file on Drive. We define outliers as observations that
+-- fall below Q1 − 1.5 IQR or above Q3 + 1.5 IQR
+-- ============================================================
+
+-- Load data from CSV
+
+@price_ts = use Drive.csv
+--file=/luminesce-examples/price_time_series.csv
+enduse;
+
+-- Calculate the IQRs
+
+@iqr_data = select
+interquartile_range(price) * (1.5) as 'iqr_x1_5',

```

Comment:
ditto the comment about usage of `'` vs `"` this should be one of: `as "iqr_x1_5"`, `as [iqr_x1_5]` or `as iqr_x1_5`

Reply:
Thanks. Updated.